src/services/parseFilesAsAttachments.ts

```diff
@@ -77,6 +93,18 @@ export async function parseFilesAsAttachmentsLegacy(
   return attachments
 }
 
-export default function parseFilesAsAttachments() {
-  // TODO: Rotate blob files for new stoarge types
+export async function correctNewAttachmentOrientation(
```

Comment:
Could you update the `camera` element to use this function as well.

Reply:
Not worth doing until the annotation task imo